src/appleseed.studio/mainwindow/rendering/renderwidget.h

```diff
@@ -139,6 +145,14 @@ class RenderWidget
         const size_t            tile_y);
 
     void paintEvent(QPaintEvent* event) override;
+    void dragEnterEvent(QDragEnterEvent* event) override;
+    void dragMoveEvent(QDragMoveEvent* event) override;
+    void dropEvent(QDropEvent* event) override;
+
+signals:
```

Comment:
Half-indent (2 spaces), like `public` etc.

Reply:
Signals are part of the public API, move that section before the private section.